solc/CommandLineInterface.cpp

```diff
@@ -322,7 +322,7 @@ void CommandLineInterface::handleGasEstimation(string const& _contract)
 	if (eth::AssemblyItems const* items = m_compiler->assemblyItems(_contract))
 	{
 		Gas gas = GasEstimator::functionalEstimation(*items);
-		u256 bytecodeSize(m_compiler->runtimeObject(_contract).bytecode.size());
+		u256 bytecodeSize(m_compiler->object(_contract).bytecode.size());
```

Comment:
True, found it in the YP:

> If the initialization code completes successfully, a fi-
nal contract-creation cost is paid, the code-deposit cost,
c, proportional to the size of the created contract’s code:
(96) c ≡ Gcodedeposit × |o

`codeDepositCost` seems appropriate then to keep in line with the YP.